README.md

```diff
@@ -532,6 +552,30 @@ Breaking changes are changes that could break a client that is relying on the Op
 [See some examples of breaking and non-breaking changes](breaking-changes.md).  
 Note: this is a Beta feature. Please report issues.
 
+### Breaking Changes Checks
+Am improved method for detecting breaking changes in specifications. It works differently form the original "Breaking changes" feature.
+
+There are multiple advantages using this feature in comparison with the original implementation:
+- output in human readable format.
+- supports 'composed' mode: outputs the API and the files where breaking changes were found.
+- has two levels of check errors: WARN - for important changes which should be noticed by developers but may not be confirmed programmatically and ERR - for most of breaking changes which should be avoided.
+- allows ignoring breaking changes through dedicated config files (one for WARNs, another for ERRs) for certain changes (e.g., an important breaking change which cannot be avoided, or false positive breaking change errors).
+- supports localization for error messages and ignored changes.
+- the set of checks can be modified by developers with their own specific checks by adding/removing checks from the slice of checks.
+- support x-stability-level extension for APIs which allows ignore breaking changes for unstable APIs. There are 4 levels draft->alpha->beta->stable. Any changes for APIs with the level draft ot alpha are allowed. Stability level may be increased, but not decreased. If there is no specified stability level for an API, breaking changes are disallowed, but the stability level can be set to any level if it is unset.
+- fewer false positive errors by design.
+- better support for type changes checks: allows changing integer->number for json/xml properties, allows changing parameters (e.g. query/header/path) to type string from number/integer/etc.
+- allows removal of responses with non-success codes (e.g. 503, 504, 403).
+- allows adding new content-type to request (with the kept current).
+- easier to extend and customize
+- will continue to be improved 
+
+Current limitation (going to be fixed in the nearest future):
+- there are no checks for `context` instead of `schema` for request parameters
+- there are no checks for `callback`s
+- not fixed false positive breaking change error when the path parameter renamed both in path and in parameters section to the same name, this can be mitigated with the checks errors ignore feature
+- doesn't support Path Prefix Modification, this can be mitigated with checks errors ignore feature 
+
 ### Non-Breaking Removal of Deprecated Resources
```

Comment:
This section should be amended to explain stability levels.

Reply:
Added section about it